site.config.ts

```diff
@@ -21,9 +21,9 @@ export default siteConfig({
   //twitter: '#',
   github: 'conrad1451',
   linkedin: 'conrad-hansen-quartey',
-  // mastodon: '#', // optional mastodon profile URL, provides link verification
-  // newsletter: '#', // optional newsletter URL
-  // youtube: '#', // optional youtube channel name or `channel/UCGbXXXXXXXXXXXXXXXXXXXXXX`
+  mastodon: '#', // optional mastodon profile URL, provides link verification
```

Comment:
I learned that uncommenting the social media handles and leaving the "#" will not just direct to the site, but crash the whole program.